src/theme/index.hbs

```diff
@@ -20,6 +20,14 @@
         <link rel="stylesheet" href="highlight.css">
         <link rel="stylesheet" href="tomorrow-night.css">
 
+        {{#previous}}
+	<link rel="prev" href="{{link}}#content">
```

Comment:
I have a feeling that this should be connected to https://github.com/azerupi/mdBook/pull/228/files#diff-9f0d373829e789d9a6923dd290c5b174R83